pint_server/database.py

```diff
@@ -213,3 +214,11 @@ def init_db(dbconfig=None, outputfile=None, echo=None,
         Base.metadata.create_all(bind=engine)
 
     return db_session
+
+
+def get_psql_server_version(db_session):
+    """Return the psql server version"""
+    result = db_session.execute("select version()")
+    for row in result:
+        version = re.search('(PostgreSQL)?[0-9]+.[0-9]+', repr(row))
```

Comment:
Few tips:
* since you are using Python `re` you can just use `\d` instead of `[0-9]`
* a regexp string should be an r-string, i.e. `r'...'` so that you don't need to do double backslashes to specify a backslash in the string

Also, is there a reason for using `repr(row)` rather than `str(row)` or just searching in `row` directly?

Reply:
using row directly does not work, i can use str.  Let me switch to r' and \d

Reply:
fixed

Reply:
fixed